molgenis-core-ui/src/main/java/org/molgenis/ui/I18nStringsPopulator.java

```diff
@@ -46,7 +47,7 @@ public void onApplicationEvent(ContextRefreshedEvent event)
 	{
 		LOG.info("Importing i18n strings...");
 
-		File file = ResourceUtils.getFile("i18n.xlsx");
+		File file = new File(Resources.getResource("i18n.xlsx").getPath());
```

Comment:
Still doesn't work, I've reopened the issue and added the log. I think you cannot instantiate a FileRepositoryCollection on top of a resource, probably?